cli/script/command-executor.ts

```diff
@@ -782,14 +782,26 @@ function getPackageMetricsString(packageObject: PackageWithMetrics): string {
     var activePercent: number = packageObject.metrics.totalActive
         ? packageObject.metrics.active / packageObject.metrics.totalActive * 100
         : 0.0;
-    var percentString: string = (activePercent === 100.0 ? "100" : activePercent.toPrecision(2)) + "%";
-    var numPending: number = packageObject.metrics.downloaded - packageObject.metrics.installed - packageObject.metrics.failed;
-    var returnString: string = chalk.green("Active: ") + percentString + " (" + packageObject.metrics.active.toLocaleString() + " of " + packageObject.metrics.totalActive.toLocaleString() + ")\n" +
-        chalk.green("Installs: ") + packageObject.metrics.installed.toLocaleString();
         
-    if (numPending > 0) {
-        returnString += " (" + numPending.toLocaleString() + " pending)";
-    }    
+    var percentString: string;
+    if (activePercent === 100.0) {
+        percentString = "100%";
+    } else if (activePercent === 0.0) {
+        percentString = "0%";
+    } else {
+        percentString = activePercent.toPrecision(2) + "%";
+    }
+    
+    var numPending: number = packageObject.metrics.downloaded - packageObject.metrics.installed - packageObject.metrics.failed;
+    var returnString: string = chalk.green("Active: ") + percentString + " (" + packageObject.metrics.active.toLocaleString() + " of " + packageObject.metrics.totalActive.toLocaleString() + ")";
+    if (showInstalls){
+        returnString += "\n" + chalk.green("Installs: ") + packageObject.metrics.installed.toLocaleString();
+        if (numPending > 0) {
+            returnString += " (" + numPending.toLocaleString() + " pending)";
+        }
+    } else if (numPending > 0) {
+        returnString += "Pending: " +  numPending.toLocaleString();
```

Comment:
I'd be in favor of always showing this number. That way, `deployment ls` always shows the number of active and pending installs, even if one of them is zero. That said, if we always show it, we would just want to not show a negative number, so it might make sense to do `Math.max(numPending, 0).toLocaleString` or something.